src/platform/openbsd.rs

```diff
@@ -121,8 +138,51 @@ impl Platform for PlatformImpl {
         unix::networks()
     }
 
-    fn network_stats(&self, _interface: &str) -> io::Result<NetworkStats> {
-        Err(io::Error::new(io::ErrorKind::Other, "Not supported"))
+    fn network_stats(&self, interface: &str) -> io::Result<NetworkStats> {
```

Comment:
Hm, any particular reason to use a custom `getifaddrs` reader instead of `platform::unix::networks`?

Reply:
`platform::unix::networks` doesn't keep fields we need for stats, I don't knew any way of reusing `platform::unix::networks` without changing struct `Network` and breaking all other implementation that uses it

Reply:
huh, right, it is quite the non-standardized API.